FlowersStore/TestePeriphery.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  TestePeriphery.swift
+//  FlowersStore
+//
+//  Created by Leonir Deolindo on 27/04/22.
+//
+
+import Foundation
+
+protocol Mano { // 'MyProtocol' is redundant
```

Comment:
<!--
  0 failure: 
  1 warning:  Protocol 'Mano' i...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/TestePeriphery.swift;
  Line: 10;
-->

- :warning: Protocol 'Mano' is unused